src/pages/GameProfile.vue

```diff
@@ -20,82 +25,144 @@
           />
         </svg>
       </router-link>
-      <h1 class="text-5xl font-bold">{{ gameDetails.name }}</h1>
-      <p class="opacity-75 mb-5">Release Date: {{ dateReleased }}</p>
-      <p
-        class="text-4xl font-semibold border-solid border-4 border-blue-600 w-24 text-center"
+      <h2
+        class="text-xl font-semibold text-white sm:text-2xl sm:leading-7 md:text-4xl"
```

Comment:
@albertmir Can we increase the font-size a little bit on md?

Reply:
Sure. Setting as **5xl**.